Can't send tokens with different wallets and nodes (probably JSON problem)

## Bug description

Since March 15th in the evening, I have JSON errors when I try to send tokens, with the cli-wallet but also with the pollen-wallet. I tried several nodes with the same outcome. 

There are others who are also having this issue. 

After updating to 0.5.1 yesterday, I was at first still able to send tokens around with the same wallets.

cli: 
```
ERROR:
  unable to read error from response body: invalid character '{' after top-level value
```

Pollen:
```
The application is not able to complete the request, due to the following error: Unexpected token { in JSON at position 116
```

## GoShimmer version

- GoShimmer:     0.5.1
- cli-wallet:          0.5.1
- Pollen-Wallet:   0.5.0

## Hardware specification

Node:
- Operating system: Ubuntu 20.04
- RAM:                      4gb
- Number of cores:   2 vCPU
- Device type:           Hetzner CX21

Wallets:
- Two different systems with Win10

## Steps To reproduce the bug

- Try to send funds with pollen-wallet or cli-wallet over a 0.5.1 GoShimmer node. 
Ask me for more info if necessary (Node IP, Addresses).

## Expected behaviour

- Being able to send tokens to another address.

## Actual behaviour

- Mentioned errors get shown.
- Affected adresses can still get (colored) tokens and show up in the wallet.


## Errors

cli-wallet
`cli-wallet send-funds -amount 25000 -dest-addr ´some address´ -access-mana-id [my node's identity] -consensus-mana-id [my node's identity]`

![image](https://user-images.githubusercontent.com/15385922/111276435-a127de80-8637-11eb-9ee8-2600bdaf95c1.png)

Pollen Wallet
![photo_2021-03-16_09-13-46](https://user-images.githubusercontent.com/15385922/111276776-febc2b00-8637-11eb-8986-40d7ed0d55e2.jpg)

this happening on 0.5.2 also, upon sending tokens. each failure generates this error in the log:

`2021-03-17T16:29:11-07:00       ERROR   MessageLayer    node/plugin.go:74       error in MessageFactory: tips could not be selected: not able to make sure that all inputs are in the past cone of selected tips
github.com/iotaledger/hive.go/node.(*Plugin).LogError
        /root/go/pkg/mod/github.com/iotaledger/hive.go@v0.0.0-20210317185122-1ac328c4054f/node/plugin.go:74
github.com/iotaledger/goshimmer/plugins/messagelayer.configure.func1
        /opt/goshimmer/plugins/messagelayer/plugin.go:51
github.com/iotaledger/hive.go/events.ErrorCaller
        /root/go/pkg/mod/github.com/iotaledger/hive.go@v0.0.0-20210317185122-1ac328c4054f/events/types.go:8
github.com/iotaledger/hive.go/events.(*Event).Trigger
        /root/go/pkg/mod/github.com/iotaledger/hive.go@v0.0.0-20210317185122-1ac328c4054f/events/event.go:32
github.com/iotaledger/goshimmer/packages/tangle.(*Tangle).Setup.func1
        /opt/goshimmer/packages/tangle/tangle.go:98
github.com/iotaledger/hive.go/events.ErrorCaller
        /root/go/pkg/mod/github.com/iotaledger/hive.go@v0.0.0-20210317185122-1ac328c4054f/events/types.go:8
github.com/iotaledger/hive.go/events.(*Event).Trigger
        /root/go/pkg/mod/github.com/iotaledger/hive.go@v0.0.0-20210317185122-1ac328c4054f/events/event.go:32
github.com/iotaledger/goshimmer/packages/tangle.(*MessageFactory).IssuePayload
        /opt/goshimmer/packages/tangle/messagefactory.go:86
github.com/iotaledger/goshimmer/packages/tangle.(*Tangle).IssuePayload
        /opt/goshimmer/packages/tangle/tangle.go:116
github.com/iotaledger/goshimmer/plugins/webapi/value.sendTransactionHandler.func1
        /opt/goshimmer/plugins/webapi/value/sendtransaction.go:75
github.com/iotaledger/goshimmer/plugins/messagelayer.AwaitMessageToBeBooked.func2
        /opt/goshimmer/plugins/messagelayer/plugin.go:167
`

--- a/client/lib.go
+++ b/client/lib.go
@@ -103,7 +103,7 @@ func interpretBody(res *http.Response, decodeTo interface{}) error {
 
 	errRes := &errorresponse{}
 	if err := json.Unmarshal(resBody, errRes); err != nil {
-		return fmt.Errorf("unable to read error from response body: %w", err)
+		return fmt.Errorf("unable to read error from response body: %w repsonseBody: %s", err, resBody)
 	}
 
 	switch res.StatusCode {

--- a/plugins/webapi/value/sendtransaction.go
+++ b/plugins/webapi/value/sendtransaction.go
@@ -73,11 +73,7 @@ func sendTransactionHandler(c echo.Context) error {
 	}
 
 	issueTransaction := func() (*tangle.Message, error) {
-		msg, e := messagelayer.Tangle().IssuePayload(tx)
-		if e != nil {
-			return nil, c.JSON(http.StatusBadRequest, SendTransactionResponse{Error: e.Error()})
-		}
-		return msg, nil
+		return messagelayer.Tangle().IssuePayload(tx)
 	}
 
 	if _, err := messagelayer.AwaitMessageToBeBooked(issueTransaction, tx.ID(), maxBookedAwaitTime); err != nil {

